packaging/nuget/nservicebus.core.nuspec

```diff
@@ -18,7 +18,9 @@
       <frameworkAssembly assemblyName="System" targetFramework="net452" />
       <frameworkAssembly assemblyName="System.Core" targetFramework="net452" />
     </frameworkAssemblies>
-    <dependencies/>
+    <dependencies>
+      <dependency id="System.ValueTuple" version="4.3.0" />
```

Comment:
👎 for this with no other reason than better safe than sorry? (or are we 100% sure this won't impact users in any way?)